extension/package.json

```diff
@@ -46,11 +46,53 @@
                 }
             ]
         },
+        "menus": {
+            "view/title": [
+                {
+                    "command": "java.test.explorer.run",
+                    "when": "view == testExplorer",
+                    "group": "navigation@0"
+                },
+                {
+                    "command": "java.test.explorer.debug",
+                    "when": "view == testExplorer",
+                    "group": "navigation@1"
+                }
+            ],
+            "view/item/context": [
+                {
+                    "command": "java.test.explorer.run",
+                    "when": "view == testExplorer",
+                    "group": "testExplorer@0"
```

Comment:
Please replace all the icons with a placeholder when the final design is not available.